fluentui_listitem/src/main/java/com/microsoft/fluentui/tokenized/tabItem/TabItem.kt

```diff
@@ -274,14 +283,22 @@ fun TabItem(
                 )
                 {
                     Box(
-                        modifier = Modifier
+                        modifier = if(indicatorColor != null) {
```

Comment:
this condition wont be needed. Just use the exisitng implementation and add the color form the indicator color token

Reply:
This is needed because of function overloading of the Modifier.background function. There are two different implementations of the background function being called.
If the background is a Color object, then we use the implementation which takes Color as an argument, if the background is a Brush object, then we use the implementation which takes Brush as an argument.

Reply:
Once token is used here then it will be Brush and not Color